tests/test_bootstrap_field_radio_select.py

```diff
@@ -24,6 +24,31 @@ class DisabledSelectTestForm(forms.Form):
     )
 
 
+class RadioSelectWithDisabledOptions(forms.RadioSelect):
+
+    def __init__(self, attrs=None, choices=(), *, disabled_choices=()):
```

Comment:
The name `choices` usually refer to an iterable of 2- (or 3-) elements sequence.
```suggestion
    def __init__(self, attrs=None, choices=(), *, disabled_values=()):
```